components/Cheese/Cheese.jsx

```diff
@@ -8,6 +8,7 @@ import styles from './Cheese.css';
 // import CheeseButton from './CheeseButton';
 // import RaisedButton from 'material-ui/FlatButton';
 import ProgressBar from '../ProgressBar';
+import palette from '../palette.css';
```

Comment:
I think our convention is to call the import `styles` and import from the local css file (which imports palette).